src/commands/start.js

```diff
@@ -56,7 +56,9 @@ function execute(collectorName, refocusUrl, accessToken) {
     debug('Exiting start.execute');
   })
   .catch((err) => {
-    throw new errors.RegistrationError('', `POST ${path} failed: ${err.message}`);
+    logger.error('Failed to register a new collector with the provided refocus instance');
```

Comment:
I'm not sure "Failed to register" is necessarily correct here anymore, since it may already be registered and is just starting. How about just "Unable to start collector XXX for Refocus instance YYY.", then the line with err.status and err.message, which I hope will provide adequate detail to fix whatever prevented it from starting?

Reply:
Good point. Is there anything wrong with just changing the word "register" to "start"?

Reply:
No, nothing wrong with that. That's fine.

Reply:
ok. updated.